eng/Packages.Data.props

```diff
@@ -112,6 +114,15 @@
     <PackageReference Update="Microsoft.Extensions.Options" Version="2.1.0" />
     <PackageReference Update="System.Threading.Channels" Version="4.6.0-preview7.19362.9" />
     <PackageReference Update="System.Text.Json" Version="4.6.0-preview7.19362.9" />
-    <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="4.6.0-preview7.19362.9" />
+   <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="4.6.0-preview7.19362.9" />   
+
+   <PackageReference Update="Azure.Core" Version="1.0.0-preview.6" />
```

Comment:
Azure.Core is already in the list and shouldn't be listed. I don't think any changes are needed to this file.